src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class BalanceOperation implements OperationHandler {

```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class implementation.